src/test/Test.hs

```diff
@@ -26,10 +26,11 @@ import Print (prettyBehaviour)
 import SMT
 import Syntax.Annotated
 
-import Debug.Trace
 import Text.Pretty.Simple
 import Data.Text.Lazy as T (unpack)
 
+import Debug.Trace
```

Comment:
can we remove this please

Reply:
Actually it was already here because there are some tracing functions that use it at the end of the file. They are not used however, so we could delete them.

https://github.com/ethereum/act/blob/121353619ca740c0f2c02dd24cc156e09e557672/src/test/Test.hs#L276-L283

Reply:
ah, sorry didn't realise this was test code, think it's fine to leave them in